grisera_api/activity/activity_service_graphdb.py

```diff
@@ -68,19 +74,18 @@ def get_activity(self, activity_id: int):
         if get_response["labels"][0] != "Activity":
             return NotFoundByIdModel(id=activity_id, errors="Node not found.")
```

Comment:
The `id` field of the `NotFoundByIdModel` class is of type `int`, while `activity_id` can be either `int` or `string`.

Reply:
Fixed